ui/guest/src/utils/contentType.ts

```diff
@@ -71,13 +72,30 @@ export function getDefaultValue(field: ContentTypeField): string {
   } else {
     switch (field.type) {
       case 'image':
-        // TODO use validation width, height
-        return 'https://via.placeholder.com/150';
+        // since it's a string, empty string needs to be validated
+        const width = JSON.parse(field.validations.width?.value).min
+          ? JSON.parse(field.validations.width?.value).min
+          : 150;
+        const height = JSON.parse(field.validations.height?.value).min
+          ? JSON.parse(field.validations.height?.value).min
+          : width;
+        return `https://via.placeholder.com/${width}x${height}`;
```

Comment:
What's a string?
By this time should be whatever it must be. Parsing should go to the service.

Reply:
Done parsing in the service. Also, new fields for minWidth, maxWidth, minHeight and maxHeight